static/js/drafts.js

```diff
@@ -101,6 +101,30 @@ export const draft_model = (function () {
         save(drafts);
     };
 
+    exports.getDraftsIdByStreamAndTopic = function (stream, topic) {
+        const drafts = this.get();
+        return Object.keys(drafts)
+            .filter(
+                (draft_id) =>
+                    drafts[draft_id].type === "stream" &&
+                    drafts[draft_id].stream === stream &&
+                    drafts[draft_id].topic === topic,
+            )
+            .sort((draft_a, draft_b) => drafts[draft_a].updatedAt - drafts[draft_b].updatedAt);
+    };
+
+    exports.getDraftsIdByRecipients = function (private_message_recipient) {
+        const drafts = this.get();
+        return Object.keys(drafts).filter(
+            (draft_id) =>
+                drafts[draft_id].type === "private" &&
+                _.isEqual(
+                    drafts[draft_id].private_message_recipient.split(",").sort(),
+                    private_message_recipient.split(",").sort(),
+                ),
+        );
```

Comment:
Why does this one not sort like the previous one does?

Reply:
I didn't write this commit, but it seems like it should also sort. Thanks for calling that out, I'll add it in!

Reply:
my bad for not going over the old commits very thoroughly before putting it up for review